__tests__/angular-json/lumberjack-mixed-angular-json.ts

```diff
@@ -0,0 +1,59 @@
+import {AngularJson} from '../../src/types/angular-json';
+
+export const lumberjackMixedAngularJson: AngularJson = {
```

Comment:
suggestion: Find a better name to this variable or at least a comment describing what *mixed* means. We're going to have a lot of test data in a matter of a few more features and we need to differentiate between the test configurations since you prefer to test this way.

Reply:
I'm open to change the way tests are written. We can discuss that in a separate issue.